client/src/components/CreateQuestion.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+
+const CreateQuestion = () => {
+  // const []
+  return (
+    <>
+      <form className="w-[70%]  border-gray100 border-4   mt-10 py-6 px-10 rounded-lg mx-auto">
+        <div className="flex flex-col">
+          <h3 className=" text-2xl font-semibold">
+            <span className="text-2xl text-orange">•</span> Create New Question
```

Comment:
Font family doesn't match the figma font design, please change.

Reply:
> I've added the dropdown, +, changed the colors of the box and input text.

@Omeshcoding 
Looks like it's almost there. Only thing I have to add is that on the Create New Question page the second section is for question _answer_. The placeholder text should read "type your answer here..." and the border should be green instead of the same blue as the question input box.

![image](https://github.com/Nim-droid-droid/anki-banki/assets/19908880/cd69ed5b-1baa-4552-959b-d646a3ba0cae)

vs the figma

![image](https://github.com/Nim-droid-droid/anki-banki/assets/19908880/c849ce6a-a79c-4af4-a3b9-99109b2adc96)

Reply:
Look like I've to change some of the font-weights in buttons as well. Thank you for the feedback.